lib/collection/src/operations/types.rs

```diff
@@ -314,7 +314,7 @@ pub struct ScrollRequestInternal {
     /// Select which payload to return with the response. Default: All
     pub with_payload: Option<WithPayloadInterface>,
```

Comment:
Nitpicking on unchanged lines: default for scroll is `true`, while other endpoints is `false`, not `All` and `None`